library/unwind/src/lib.rs

```diff
@@ -3,8 +3,9 @@
 #![feature(link_cfg)]
 #![feature(staged_api)]
 #![cfg_attr(not(target_env = "msvc"), feature(libc))]
+#![cfg_attr(bootstrap, allow(unexpected_cfgs))]
 #![cfg_attr(
-    all(target_family = "wasm", not(target_os = "emscripten")),
+    all(target_family = "wasm", any(not(target_os = "emscripten"), emscripten_wasm_eh)),
```

Comment:
The cfg must also be gated behind a feature gate, please create one in https://github.com/rust-lang/rust/blob/f55a5d2e067629c7939a7d8373e19dc673552934/compiler/rustc_feature/src/unstable.rs#L183 and add it in https://github.com/rust-lang/rust/blob/f55a5d2e067629c7939a7d8373e19dc673552934/compiler/rustc_feature/src/builtin_attrs.rs#L18
and then add the feature gate where necessary.

Reply:
I think I did the right stuff to add the feature, please let me know if anything is missing.